src/resources/postcss/views/list/_event.pcss

```diff
@@ -30,20 +30,42 @@
 		}
 	}
 
+	.tribe-events-calendar-list__event-row--featured {
+
+		.tribe-events-calendar-list__event-date-tag-datetime {
+			border-left: 3px solid var(--color-accent-primary);
+
+			@media (--viewport-medium) {
+				border-left: none;
+				border-right: 2px solid var(--color-accent-primary);
+				box-shadow: 2px 0 0 0 var(--color-accent-primary);
```

Comment:
curious, why are we using a box shadow here rather than using a border-right of 4px (or even an `&:after` pseudoelement)?

Reply:
I made that split, because in the design the border is is positioned in two different parts: https://cloudup.com/cLNgfzxICpA